### PR TITLE
docs(contributions) - Contribution file added/linked to from readme

### LINKED ISSUE #8: creating a contributions file
Expanding contributions into its own file, and expanding on how to properly add features. Key considerations:

- People interested in contributing should check existing issues first
- Pull requests should be attached to an accompanying issue. If your PR doesn't address any issue currently listed, make one!
- Links to easy to parse resourced for creating issues and pull requests

### PR DESCRIPTION
**Pull request by tkshill**
Tried to create a basic template that a beginner could follow along with using appropriate resources. Closes #8

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+If you feel like there's a contribution you would like to make that isn't represented by an already existing issue, feel free to create your own!
+
+## Step 4. Creating a PR
+At this point, you have two options.
+
+**The first option** is using the github interface to fork the repo and submit a Pull Request (no code or terminals or IDE required). [This guide](https://guides.github.com/activities/hello-world/) shows just how to do that for a small personal repo. You would simply replace the step of creating a new repository to just navigating to the virtual coffee open source one and forking that instead. This is a great idea if you simply plan to add to or edit one of the markdown files we use for documentation in this project.
+
+**The second option** is go the traiditional route of forking the repo, creating a local copy of that fork, and working on your changes that way. This is also the only way to go if this project expands to include an associated application. For that we recommend [this guide](https://www.dataschool.io/how-to-contribute-on-github/).
```
Comment (cambardell) [inline CONTRIBUTING.md:44]: Typo here in traditional

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+This section is just little notes and bits of info that can smooth over some of the bumps and hiccups that can come along with contributiong.
+
+- While this isn't absolutely required, we highly recommend **associating your Pull Requests with the Issue that they're intended to address**. This makes review much easier and avoids confusion when looking back at past commits. Github allows you to link a PR to an issue both during and after the PR's creation (the option should located in rightmost panel of the github PR interface).
+- **Please do not hesitate to ask for help** in any part of this process if you feel confused. As soon as they can, project maintainers can try and get you through the parts which are confusing you. Just be aware that no one here is a github expert :), we're just folks happy and willing to help others get some experience.
+- Be extra careful when working with git in the command line. Incorrectly typed names or commands can have strange results, and navigating git issues can be very perplexing. Fortunately, there's a plethora of resources on fixing said issues, and rest assured that any error you make has already been done, and solved, by someone already
```
Comment (cambardell) [inline CONTRIBUTING.md:62]: Already is here twice, and there's no period.

**Comment by tkshill** [comment]
@saramccombs @healeyb Anyone who's able to give this a look over/merge, I'd appreciate it. :)

**Comment by saramccombs** [comment]
I’ll take a look as soon as I get back to my desk. ❤️

**Pull request merged**